api/resource_postgres.go

```diff
@@ -185,6 +185,25 @@ func (client *Client) ListPostgresUsers(ctx context.Context, appName string) ([]
 	return *data.App.PostgresAppRole.Users, nil
 }
 
+func (client *Client) EnablePostgresConsul(ctx context.Context, appName string) (*PostgresEnableConsulPayload, error) {
+	query := `
```

Comment:
this should probably be a `const`ant, no?

Reply:
I am just following the pattern this file and others are using.

Reply:
It's a legacy thing and I get it. Not going to insist on this